modules/ossm-enabling-sidecar-injection.adoc

```diff
@@ -0,0 +1,141 @@
+// Module included in the following assemblies:
+// install/ossm-sidecar-injection
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-enabling-sidecar-injection_{context}"]
+= Enabling sidecar injection
+
+The sidecar injection process in {istio} uses label-based rules to determine if a pod should be injected with a sidecar. If a namespace has both the `istio-injection` label and the `istio.io/rev` label, the `istio-injection` label takes precedence, treating the namespace as part of the default revision.
+
+The sidecar injector follows the following logic:
+
+* If either the `istio-injection` label or the `sidecar.istio.io/inject` label is set to disabled, the pod is not injected.
+* If any of the following labels are enabled the pod is injected with a sidecar: `istio-injection`, `sidecar.istio.io/inject`, or `istio.io/rev`.
+
+.Prerequisites
+
+* The {SMProduct} operator is installed
+* An {istio} Container Network Interface (CNI) resource is created
+
+.Procedure
+
+. Create namespace called `istio-system`: 
++
+[source,terminal]
+----
+$ oc create ns istio-system
```

Comment:
Should this be `oc new-project` instead of `oc create ns`?

Reply:
@luksa I am not sure which is correct. Changing it to: `oc new-project`